views/graph.py

```diff
@@ -34,27 +35,39 @@ def _translate(context, text, disambig):
 
 
 class GraphPlot(pg.PlotWidget):
+    """Graph element"""
 
     def __init__(self, parent=None):
+        "Constructor"

```

Comment:
a) driedubbele quotes
b) `__init__` is altijd een constructor, dus dat hoeft er niet bij.